docs/sections/user_guide/yaml/components/ww3.rst

```diff
@@ -0,0 +1,24 @@
+.. _ww3_yaml:
+
+ww3
+===
+
+Structured YAML to configure WaveWatchIII as part of a compiled coupled executable is validated by JSON Schema and requires the ``ww3:`` block, described below.
+
+Here is a prototype UW YAML ``ww3:`` block, explained in detail below:
+
+.. highlight:: yaml
+.. literalinclude:: ../../../../shared/ww3.yaml
+
+UW YAML for the ``ww3:`` Block
+------------------------------
+
+namelists:
```

Comment:
```suggestion
namelist:
```

In the shared ww3.yaml the block is called `namelist:`

Reply:
I knew I was gonna miss one when I changed that, thanks!